MSAL/src/requests/MSALBaseRequest.m

```diff
@@ -256,44 +270,34 @@ - (void)acquireToken:(nonnull MSALCompletionBlock)completionBlock
              return;
          }
          
-         NSError *cacheError = nil;
-         MSALTokenCache *cache = self.parameters.tokenCache;
-         
-         MSALAccessTokenCacheItem *atItem = [cache saveAccessTokenWithAuthority:_parameters.unvalidatedAuthority
-                                                                       clientId:_parameters.clientId
-                                                                       response:tokenResponse
-                                                                        context:_parameters
-                                                                          error:&cacheError];
-         
-         if (!atItem)
+         NSError *msidError = nil;
+         MSIDAADV2TokenResponse *msidTokenResponse = [[MSIDAADV2TokenResponse alloc] initWithJSONDictionary:tokenResponse.jsonDictionary error:nil];
```

Comment:
nit: response can be generated by a strategy

Reply:
done